control/control.openSUSE.xml

```diff
@@ -302,7 +302,7 @@ textdomain="control"
                 <min_size config:type="disksize">1 GiB</min_size>
                 <max_size config:type="disksize">2 GiB</max_size>
                 <weight config:type="integer">10</weight>
-                <adjust_by_ram config:type="boolean">true</adjust_by_ram>
+                <adjust_by_ram config:type="boolean">false</adjust_by_ram>
```

Comment:
Because on systems with 32gb of RAM or higher (which are not uncommon) it can quite easily take minutes to hibernate

On systems with 128 gb ssds (not uncommon) we are taking away a significant proportion of a users disk space for a feature they may never use.

I might have a different opinion if we had hybrid suspend or other such fanciness, but we don’t, so we shouldn’t be wasting users disk space and/or time by default.